service/src/main/kotlin/io/provenance/api/domain/usecase/objectStore/store/models/StoreObjectRequest.kt

```diff
@@ -2,16 +2,14 @@ package io.provenance.api.domain.usecase.objectStore.store.models
 
 import io.provenance.api.models.account.AccountInfo
 import io.provenance.api.models.p8e.PermissionInfo
-import java.security.PrivateKey
-import java.security.PublicKey
+import io.provenance.scope.encryption.model.KeyRef
 
 data class StoreObjectRequest(
     val bytes: ByteArray,
     val type: String?,
     val objectStoreUrl: String,
     val useObjectStoreGateway: Boolean,
-    val publicKey: PublicKey,
-    val privateKey: PrivateKey,
+    val keyRef: KeyRef,
```

Comment:
Should we label this PR as minor? Will other endpoint request bodies, like those for contract execution, be updated in the same fashion as well?

Reply:
ya, that's a good idea. 

All of the updates needed are applied in this PR. Contract execution resolves all needed key material inside the usecase itself (unlike the store object request as you've pointed out). in this case, the callers of the StoreObject use case are for protos / files and each manually create the request object needed here.